web-client/Cargo.toml

```
@@ -21,12 +21,18 @@ maintenance = { status = "experimental" }
 crate-type = ["cdylib"]
 
 [dependencies]
+beserial = { path = "../beserial", features = ["derive"] }
 futures = { package = "futures-util", version = "0.3" }
 gloo-timers = "0.2"
 log = { package = "tracing", version = "0.1", features = ["log"] }
 wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
 
+nimiq-hash = { path = "../hash" }
+nimiq-blockchain-interface = { path = "../blockchain-interface" }
+nimiq-blockchain-proxy = { path = "../blockchain-proxy", default-features = false }
+nimiq-consensus = { path = "../consensus", default-features = false }
+nimiq-network-interface = { path = "../network-interface" }
 nimiq-network-libp2p = { path = "../network-libp2p" }
```

We try to keep them alphabetically ordered:
```suggestion
beserial = { path = "../beserial", features = ["derive"] }
nimiq-blockchain-interface = { path = "../blockchain-interface" }
nimiq-blockchain-proxy = { path = "../blockchain-proxy", default-features = false }
nimiq-consensus = { path = "../consensus", default-features = false }
nimiq-hash = { path = "../hash" }
nimiq-network-interface = { path = "../network-interface" }
nimiq-network-libp2p = { path = "../network-libp2p" }
```